### ISSUE TITLE
PendingDeprecationWarning with urls

### ISSUE THREAD

**Issue by nikolaik** [python-social-auth/social-app-django#276]
After upgrading to django 3.1 I'm getting this with v4.0.0:
```
social_django/urls.py:15: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
  url(r'^login/(?P<backend>[^/]+){0}$'.format(extra), views.auth,

social_django/urls.py:17: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
  url(r'^complete/(?P<backend>[^/]+){0}$'.format(extra), views.complete,

social_django/urls.py:20: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
  url(r'^disconnect/(?P<backend>[^/]+){0}$'.format(extra), views.disconnect,

social_django/urls.py:22: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
  url(r'^disconnect/(?P<backend>[^/]+)/(?P<association_id>\d+){0}$'

**Comment by FlorentClarret** [comment]
I also got this warning with the 3.4.0 and was willing to fix it. However, this is already patched since https://github.com/python-social-auth/social-app-django/pull/269. I will just wait for the next release.

**Comment by nijel** [comment]
Closing as this is fixed by #269.

**Comment by stianjensen** [comment]
```
